velox/tpch/gen/TpchGen.cpp

```diff
@@ -125,7 +126,7 @@ Table fromTableName(std::string_view tableName) {
       fmt::format("Invalid TPC-H table name: '{}'", tableName));
 }
 
-size_t getRowCount(Table table, size_t scaleFactor) {
+size_t getRowCount(Table table, double scaleFactor) {
```

Comment:
Shouldn't we check here (or somewhere else) that this number is not negative?

Reply:
@pedroerp, I added some checks for negative values.